srtcore/core.cpp

```diff
@@ -56,6 +56,7 @@ modified by
 #include <sstream>
 #include "srt.h"
 #include "queue.h"
+#include "api.h"
```

Comment:
Here api.h must be included, core.h theoretically isn't then necessary, but it should be there for formality, as this is the header for this very file.